package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "microplan",
-  "version": "1.4.2",
+  "version": "1.5.0",
```

Comment:
There is a recently merged [change](https://github.com/microplan-xyz/microplan/pull/83/files)  which breaks the module. So, we should bump the major version.
Not sure if this is the right time to publish as few [tasks](https://github.com/microplan-xyz/microplan/issues/78) are still pending.
Lets hear @scriptnull inputs.

Reply:
That's fine. We recently removed `autopublish` to npm. I will merge this PR with this change.  We will increment the major version after pushing few more changes and go for release. Until that, let the version be this.